.github/workflows/auto-job-v2.yaml

```diff
@@ -0,0 +1,29 @@
+name: auto-job-v2
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
do we need to send trigger command for updating action?

Reply:
no, for updating, we can do it manually

Reply:
I prefer e2e can be triggered automatically when update

Reply:
maybe add `synchronize` ?

Reply:
synchronize added